api/src/certification/enrolment/domain/models/UserEligibilityList.js

```diff
@@ -0,0 +1,52 @@
+import { LABEL_FOR_CORE } from './UserEligibilityCalculator.js';
+
+export class UserEligibilityList {
```

Comment:
Pinaillage 

```suggestion
export class UserEligibilities {
```

ou bien sinon, car ca contient au final plusieurs listes

```suggestion
export class UserEligibilityLists {
```